recipes/rastervision-core/0001-fix-imports.patch

```diff
@@ -0,0 +1,33 @@
+diff --git a/requirements.txt b/requirements.txt
+index e69de29..140249a 100644
+--- a/requirements.txt
++++ b/requirements.txt
+@@ -0,0 +1,11 @@
++rastervision_pipeline
++numpy<1.22
++shapely==1.6.*
++pillow==8.3.*
++pyproj==2.6.*
++rasterio==1.2.6
++scikit-learn==0.24.*
++imageio==2.3.*
++pystac==0.5.6
++mask-to-polygons==0.0.2
++opencv-python
+\ No newline at end of file
```

Comment:
I found two different requirements folders for this repo
- one in the main rastervision repo https://github.com/psortos/staged-recipes/tree/add-rastervision-core/recipes/rastervision-core 
- another in the pypi pacakge for rastervision-core https://files.pythonhosted.org/packages/40/d0/4ff1c4c5b31b443deeffff1ce48f5bf03f979575e4b8726b121062180d8a/rastervision_core-0.13.1.tar.gz 
This requirements removes some of the version requirements which were creating conflicts